cli/pack/systemd_dir.go

```diff
@@ -13,6 +15,14 @@ import (
 
 const (
 	instanceNameSpecifier = "%i" // https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers
+
+	defaultSystemdUnitParamsFile = "systemd-unit-params.yml"
```

Comment:
Here you use ``File``, but at lines above - ``Path``

Reply:
Yes, I like it when `file` variables are different from the `path`, sometimes it is confusing. `defaultSystemdUnitParamsFileName` is a long name, but it very good - you understand that it's a name (not a path) of a file in the project root.